### PR TITLE
Add the Release Notes for VMware vCenter connector improvement

### PR DESCRIPTION
**Pull request by jbremond**

### CODE REVIEW

File: admin/en/source/pages/changelog.rst [hunk 1]
```diff
@@ -15,6 +15,7 @@ New Features
 * Support to create appliance templates based on Windows 2016 operating system
 * Ability to scan and migrate Windows 2016 instances
 * Application and services information now captured and displayed when scanning a Windows-based instance
+* Publication to VMware vCenter improved. UForge now publishes templates to vCenter, rather than instances. The datacenter information (ESXi hypervisor, datastore, network name, etc) are automatically retrieved by UForge and prefilled for publication to VCenter.
```
Comment (jgweir) [inline admin/en/source/pages/changelog.rst]: VCenter -> VMware vCenter (should use the correct product name)

File: apis/en/source/pages/changelog.rst [hunk 2]
```diff
@@ -15,6 +15,7 @@ New Features
 * Support to create appliance templates based on Windows 2016 operating system
 * Ability to scan and migrate Windows 2016 instances
 * Application and services information now captured and displayed when scanning a Windows-based instance
+* Publication to VMware vCenter improved. UForge now publishes templates to vCenter, rather than instances. The datacenter information (ESXi hypervisor, datastore, network name, etc) are automatically retrieved by UForge and prefilled for publication to VCenter.
```
Comment (jgweir) [inline apis/en/source/pages/changelog.rst]: VCenter -> VMware vCenter (should use the correct product name)

File: end-user/en/source/pages/changelog.rst [hunk 3]
```diff
@@ -15,6 +15,7 @@ New Features
 * Support to create appliance templates based on Windows 2016 operating system
 * Ability to scan and migrate Windows 2016 instances
 * Application and services information now captured and displayed when scanning a Windows-based instance
+* Publication to VMware vCenter improved. UForge now publishes templates to vCenter, rather than instances. The datacenter information (ESXi hypervisor, datastore, network name, etc) are automatically retrieved by UForge and prefilled for publication to VCenter.
```
Comment (jgweir) [inline end-user/en/source/pages/changelog.rst]: VCenter -> VMware vCenter (should use the correct product name)

File: admin/en/source/pages/changelog.rst (hunk 1 above)
Comment (emuus) [inline admin/en/source/pages/changelog.rst]: datacenter information (...) IS

**Comment by jbremond** [comment]
Thanks @emuus @jgweir for your comment. Please find a new proposal on this PR

**Comment by AliceAlbano** [comment]
Seen with jgweir, PR is ready to be merged.

**Pull request merged**